yearn/special.py

```diff
@@ -27,6 +34,38 @@ def total_value_at(self, block=None):
         crv_price = magic.get_price(crv, block=block)
         return crv_locked * crv_price
 
+    @property
+    def strategies(self):
+        return []
+
+    def apy(self, samples: ApySamples) -> Apy:
+        curve_3_pool = Contract("0xbEbc44782C7dB0a1A60Cb6fe97d0b483032FF1C7")
+        curve_reward_distribution = Contract("0xA464e6DCda8AC41e03616F95f4BC98a13b8922Dc")
+        curve_voting_escrow = Contract("0x5f3b5DfEb7B28CDbD7FAba78963EE202a494e2A2")
+        voter = "0xF147b8125d2ef93FB6965Db97D6746952a133934"
+        graph = requests.get("https://api.coingecko.com/api/v3/simple/price?ids=curve-dao-token,vecrv-dao-yvault&vs_currencies=usd").json()
```

Comment:
Ideally have no external requests, check if `yearn.prices.magic.get_price` returns accurate enough value for you here.